server/routes/bookingController.ts

```diff
@@ -0,0 +1,26 @@
+import type { RequestHandler } from 'express'
+import { VisitService } from '../services'
+
+export default class BookingController {
```

Comment:
```suggestion
export default class BookingsController {
```

Should the name be plural 🤔 ?